cancat/scripts/cancat2candump.py

```diff
@@ -0,0 +1,18 @@
+# Command line entry point for cancat2candump
```

Comment:
i believe that what you are trying to do... wants these files to be actually:
cancat/scripts/cancat2candump/__main__.py  and then pypi (or distutils?) creates those into a script??

Reply:
I spent way too long trying to figure that out :) I think that it should work as-is. But if necessary the change isn't too big that it can't be done later if we need to.